src/RuleHandlerResolverInterface.php

```diff
@@ -4,21 +4,17 @@
 
 namespace Yiisoft\Validator;
 
-use Yiisoft\Validator\Exception\RuleHandlerInterfaceNotImplementedException;
-use Yiisoft\Validator\Exception\RuleHandlerNotFoundException;
-
 /**
- * Rule handler resolver is obtaining a resolver for a given rule class name.
+ * An interface allowing to resolve a rule handler name to a corresponding rule handler instance.
  */
 interface RuleHandlerResolverInterface
 {
     /**
-     * Obtain a resolver for a given rule class name.
+     * Resolves a rule handler name to a corresponding rule handler instance.
      *
-     * @param string $className Rule class name.
+     * @param string $name A rule handler name ({@see RuleInterface}).
      *
-     * @throws RuleHandlerNotFoundException
-     * @throws RuleHandlerInterfaceNotImplementedException
```

Comment:
Why remove `@throws` ?

Reply:
Because throwing exceptions is a part of implementation.

Reply:
Discussed it and decided to keep them.